examples/mission_basic/mission_basic.py

```diff
@@ -1,216 +1,241 @@
-#!/usr/bin/env python

```

Comment:
i think your gopro example needs its own example section, the mission_basic example isn't a good fit since not all the users are using or will use a gopro